### ISSUE TITLE
[com.google.fonts/check/ttx-roundtrip] correct rational of name id 256 

### ISSUE THREAD

**Issue by RosaWagner** [fonttools/fontbakery#3749]
**Honest curiosity, how is this check:**
🔥 FAIL: Checking with fontTools.ttx ([com.google.fonts/check/ttx-roundtrip](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/ttx-roundtrip))
🔥 FAIL name id 256 missing from name table
ℹ INFO While converting TTF into an XML file, ttx emited the messages listed below.

**not a duplicate of this one:**
⚠ WARN: Ensure Stylistic Sets have description. ([com.google.fonts/check/stylisticset_description](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/stylisticset_description))

—since adding stylistic sets name result in the addition of name ID 256 (and following ids if more).

**EDIT: the problem is that the rational of this check is confusing; the name id 256 is not missing, it is EMPTY.**
So I could suggest for now to simply update the rational with "name ID 256 is present but empty". This name ID is often dedicated to axis, style or stylistic set names.

Whatever it is missing or empty, it shouldn't FAIL for Google Font Profile.

**Comment by RosaWagner** [comment]
I also found font using this name ID to display the axis name, but then shouldn't be expected in static fonts, right?

**Comment by RosaWagner** [comment]
Yes, I did't know this name ID at first, and I saw in some VF it was used to display some axis name, so I didn't understand why it was appearing in static fonts.

**Comment by vv-monsalve** [comment]
Good catch. That ttx-nameID256 was kind of cryptic and knowing now the SS sets issue brings a final resolution to #3155

**Comment by n8willis** [comment]
I just bumped into this (the "name ID 256 missing from name table" result, from "Checking with fontTools.ttx" on Matemasie (https://github.com/google/fonts/pull/7875#issuecomment-2197135453)

And it wasn't simple to understand, since FontBakery's report does _not_ report the stylistic-set-description issue that vv-monsalve refers to, or anything else mentioned as the potential TTX triggers for it (e.g., STAT or axes). 

So, some clarification in the output would be really helpful. If the stylistic-set-descrription check has gone away, then perhaps at least a first step would be for Bakery to note what the possibilities are.
